generators/generator-base.js

```diff
@@ -1683,6 +1684,24 @@ module.exports = class extends PrivateBase {
     }
 
     /**
+     * Print an info message.
+     *
+     * @param {string} value - message to print
+     */
+    info(msg) {
```

Comment:
and I agree that we should have a uniform way to log success, error, info and debug rather than using all kind of colors with chalk. Yeoman's doc does not provide much guidance